pages/x/index.tsx

```diff
@@ -17,18 +17,22 @@ const ThirdPartyRegistryList = () => {
     setQuery(event.target.value);
   }
 
-  const list = useMemo(
+  // Sorting the list only if entries are changed
+  const list = useMemo(() => {
```

Comment:
You don't need to use `filter` separately. You could do something like below. 

```ts
const list = useMemo(
    () =>
      Object.keys(entries)
        .filter((name) => {
          const queryReg = new RegExp(
            query.split(" ").reduce((a, b) => a + ".*" + b)
          );
          return (
            name.match(queryReg) || (entries[name].desc ?? "").match(queryReg)
          );
        })
        .sort((nameA, nameB) => nameA.localeCompare(nameB)),
    [entries, query]
  )
```

Reply:
Does it make sense to sort every time query changes? Sorting algorithms are expensive compared to filter. And sorting is one time operation when entries are changed. That's the idea behind separating.

Reply:
> Does it make sense to sort every time query changes? Sorting algorithms are expensive compared to filter. And sorting is one time operation when entries are changed. That's the idea behind separating.

Sorting don't need https://github.com/denoland/deno_website2/pull/1171#discussion_r443072926
It has test case [database.test.ts#L50-L53](https://github.com/denoland/deno_website2/blob/master/util/database.test.ts#L50-L53)